docs/Doxyfile

```diff
@@ -0,0 +1,1637 @@
+# This file describes the settings to be used by the documentation system
```

Comment:
This file was in .gitignore because it is generated on build from the Doxyfile.in template with the CMake `configure_file` command. Tokens in the template are replaced with the corresponding CMake variable values.